packages/common/src/localization/strings.ts

```diff
@@ -0,0 +1,70 @@
+export function getFakeString(text: string) {
```

Comment:
rename files to fake.strings? 

it took me a minute of looking before I realized there was no actual localization string code in here.